### PR TITLE
various fixes for 030-iob18

### PR DESCRIPTION
**Pull request by hansfbaier**
Signed-off-by: Hans Baier <hansfbaier@gmail.com>

### CODE REVIEW

**Comment by hansfbaier** [comment]
One of the results of these fixes is the discovery of a new bit:
```
RIOB18.IOB_Y0.ZIBUF_LOW_PWR origin:030-iob18 39_125
```

**Comment by hansfbaier** [comment]
@tmichalak Ready for review. The Vref changes get rid of a lot of warning of mismatching Vref, but don't change the outcome. The IBUF_LOW_PWR_SUPPORTED change finds a new feature.

File: fuzzers/030-iob18/Makefile [hunk 1]
```diff
@@ -23,6 +23,8 @@ build/segbits_riob18.db: build/segbits_riob18.rdb process_rdb.py bits.dbf
 	sed '/IOB18.IOB_Y1.LVDS/d' -i build/segbits_riob18.rdb
 	python3 process_rdb.py build/segbits_riob18.rdb > build/segbits_riob18_processed.rdb
 	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --groups tag_groups.txt --seg-fn-in build/segbits_riob18_processed.rdb --seg-fn-out $@
+	echo 'RIOB18.IOB_Y0.SSTL12_SSTL135_SSTL15.IN !38_126 39_127' >> $@
```
Comment (tmichalak) [inline fuzzers/030-iob18/Makefile:26]: I can't fight the feeling that this looks like a hack. I know that this fuzzer is kintex specific, but have never had to do things like this before. I would have to think if there is a better solution for this. For the time being we can open an issue for this and add a comment linking to it.

Comment (hansfbaier) [reply fuzzers/030-iob18/Makefile:26]: I don't like it much either, unfortunately, but the python script I wrote shows that the bit 39_127 disappears as soon we have a differential input on the Y0 pads, which wipes out the candidates. Unfortunately selectively either setting the tag to 0 or removing the tag in this case caused alterations in the database which did not work on hardware. I am a bit out of ideas of how else I could deal with this, but agree with your proposal to open an issue.

File: fuzzers/030-iob18/write_io_banks.tcl [hunk 2]
```diff
@@ -11,7 +11,11 @@ open_io_design -name io_1
 set fp [open "iobanks.txt" "w"]
 foreach iobank [get_iobanks] {
     foreach site [get_sites -of $iobank] {
-        puts $fp "$site,$iobank"
+        # we only care about the high performance banks here
+        # and those are numbered 32, 33, 34...
+        if {[string match "3*" $iobank]} {
```
[the remark this answers is not in the document]
Comment (hansfbaier) [reply fuzzers/030-iob18/write_io_banks.tcl:16]: Oh you are right, I thought those bank numbers would be consistent across all 7 series devices, but I found a counterexample 
image

File: fuzzers/030-iob18/Makefile (hunk 1 above)
Comment (hansfbaier) [reply fuzzers/030-iob18/Makefile:26]: I just pushed a fix.

**Comment by hansfbaier** [comment]
@tmichalak I created the issue: https://github.com/f4pga/prjxray/issues/2073

**Comment by hansfbaier** [comment]
@tmichalak I created the issue: https://github.com/f4pga/prjxray/issues/2073

**Review by tmichalak — APPROVED** [review]
LGTM, but let's wait for green CI just in case.

**Pull request merged**